internal/session/types.go

```diff
@@ -70,4 +72,5 @@ type OIDCWorkflowSession struct {
 	TargetURI                  string
 	AuthURI                    string
 	RequiredAuthorizationLevel authorization.Level
+	Created                    int64
```

Comment:
Append Timestamp as well. It should be clear what this variable holds by simply reading the name if there is no dedicated type.